### ISSUE TITLE
Define XRelease schedule

### ISSUE THREAD

**Issue by mdoering** [CatalogueOfLife/xcol#199]
When will the XRelease be published in relation to the base release and when is data tested/reviewed?
Do we release & publish an XRelease right after the base was published?

**Comment by mdoering** [comment]
We agree to keep the weekly release builds and schedule the release following the base release as the one to be published.
@yroskov @camiplata @DianRHR could we target a fixed week or day of the month for releasing the base and subsequently the xrlease? How about the 2nd last thursday of every month to release the base and the last friday of the month for the XRelease? That would leave one friday XR build (immediately following the thursday release) to review any changes?

**Comment by mdoering** [comment]
It might be good to put these dates into a shared calendar, so we can deviate in case of holidays, sickness, travels and other impacts.

**Comment by yroskov** [comment]
Just in case, if you are talking also about Global CoL, we cannot fix a release date. The date depends on many issues, and very much on IT processes, conversions plus reliable functionality of CLB.

**Comment by mdoering** [comment]
We do, yes. We need to synchronise releases so they can both happen in the same month and the XR depends on the base to go out first. Think about how we can best coordinate this and discuss this in our next call. It can be flexible, but we have a dependency now.

**Comment by yroskov** [comment]
In this case, I would suggest to keep a last day of the month as the cap for the Global CoL release, and you can do xCoL release in the first week of the following month.

**Comment by mdoering** [comment]
could do, but then they do not share the same month in their version and probably alias which is a bit odd, no? It should be seen as a release package. Would it not be better to target the middle or 2/3 of the month?

**Comment by yroskov** [comment]
Let's have a look, what we had.

Here is a list of Global CoL releases in 2024-2025:
2024-01-24
2024-02-22
2024-03-26
2024-04-26
2024-05-20
2023-06-27
2024-07-18
2024-08-29
2024-09-25
2024-10-22
2024-11-18
2024-12-19
2025-01-17
2025-02-13

Can you please share the dates of deployments to the portal and dates of xCoL releases?

**Comment by mdoering** [comment]
good point that we also need to cater for @olafbanki review of the release. The portal deployment with news post could happen a bit later, but making the release public is a requirement for starting the XRelease. So that adds an extra day.

XRelease start every friday night latin american time and take 24-36 hours. We would want to stick with that, as it nicely runs over the weekend and the backend cannot be deployed or stopped during that time. Doing weekly XRs basically gives us 3 test runs and time for review & adjustments

**Comment by yroskov** [comment]
If so, a last day of the month as a cap for Global CoL and first Friday of the following month for xCoL should work well.

**Comment by mdoering** [comment]
but we then end up with different months for the sister releases

**Comment by yroskov** [comment]
I do not see a problem

**Comment by yroskov** [comment]
Each release has CLB ID and DOI, which have no dates at all. _xRelease 308145 is based on COL 281951_ - sound is good to me.

**Comment by mdoering** [comment]
https://www.catalogueoflife.org/data/changelog

COL25.2, the February edition, version 2025-02-13. That's what people call it. The key is technical.
It would be desirable to then also have a COL25.2 XR, the February XR edition, that clearly correlates the 2 COL releases.
It is also what is shown in all UIs and ChecklistBank forms/selectors

**Comment by mdoering** [comment]
@camiplata can we document the agreed conclusion here please?

**Comment by DianRHR** [comment]
According to the meeting in Bogotá, it was agreed that the releases of the base COL will be on the 15th of each month and that will be the base version for the XR. 

![Image](https://github.com/user-attachments/assets/9500e9e1-a08b-4028-be1b-5a9f90726cec)
